gcc/config/riscv/riscv.h

```diff
@@ -513,15 +513,15 @@ enum reg_class
    `crtl->outgoing_args_size'.  */
 #define OUTGOING_REG_PARM_STACK_SPACE(FNTYPE) 1
 
-#define STACK_BOUNDARY 128
+#define STACK_BOUNDARY (TARGET_RVE ? 32 : 128)
```

Comment:
For RVE, the stack alignment requirement is just 4 bytes.  So, this is just a performance tradeoff (should I align the stack to 16 bytes so that I can compress ADDI into C.ADDI16SP).

Fortunately, RVE stack frames should be small (there are only 3 callee-saved registers: s0, s1, ra), so usually you can use C.ADDI to adjust the stack pointer.